### PR TITLE
feat: new subcomand replay

### PR DESCRIPTION
**Pull request by zhangsoledad**
### Proposal changes
Introduce more generalize `replay` subcommand replace `prof`, replay local achieved chain data. profile or sanity check.

### CODE REVIEW

File: util/instrument/src/iter.rs [hunk 1]
```diff
@@ -51,7 +51,7 @@ impl Iterator for ChainIterator {
     fn size_hint(&self) -> (usize, Option<usize>) {
         match self.current {
             Some(ref b) => (1, Some((self.tip - b.header().number() + 1) as usize)),
-            None => (0, Some(0)),
+            None => (0, None),
```
Comment (zhangsoledad) [inline util/instrument/src/iter.rs:54]: returns (0, None) which is correct for any iterator

**Comment by doitian** [comment]
Please resolve the conflicts. Ready to merge.

**Comment by doitian** [comment]
bors r=quake,doitian

**Comment by doitian** [comment]
bors try

**Comment by doitian** [comment]
bors r=quake,doitian

**Pull request merged**